crates/admin/src/cluster_controller/cluster_state.rs

```diff
@@ -110,7 +109,7 @@ where
         get_state_router: RpcRouter<GetProcessorsState, N>,
         cluster_state_tx: Arc<watch::Sender<Arc<ClusterState>>>,
         metadata: Metadata,
-    ) -> Result<Option<JoinHandle<()>>, ShutdownError> {
+    ) -> Result<Option<TaskHandle<()>>, ShutdownError> {
```

Comment:
Could we use `spawn_unmanaged` in `start_refresh_task` as well?

Reply:
Good call.